helm/cluster-openstack/values.schema.json

```diff
@@ -214,36 +146,13 @@
                 "usernamePrefix": {
                     "type": "string"
                 }
-            },
-            "required": [
-                "clientId",
-                "groupsClaim",
-                "issuerUrl",
-                "usernameClaim"
-            ]
+            }
         },
         "organization": {
             "type": "string"
         },
         "subnetName": {
             "type": "string"
         }
-    },
-    "required": [
```

Comment:
can you add the `required` section back?

Reply:
done

Reply:
Note that we use `required` fields are not only for top level but also for subobjects.